### ISSUE TITLE
New Github app doesn't allow unsubscribing to 'statuses'

### ISSUE THREAD

**Issue by sandersiim** [integrations/slack#1175]
The old GitHub app allowed to unsubscribe from PR statuses related to checks under the PR. After upgrading to new app, the statuses information is now spammed with every opened PR notification and there doesn't seem to be any way to turn it off. Furthermore, the statuses don't even sync as checks are ultimately passed, making it completely useless spam.

How to turn PR statuses notifications off in the new app? Or alternatively, how can I revert back to the old app?

**Comment by apurva1112** [comment]
Hello @sandersiim , with the new GitHub app for slack, you cannot unsubscribe statuses exclusively as in the old app. We will take this as a feedback and look into further improvement of the user experience.

**Comment by sandersiim** [comment]
But in the current state of things, what's the point of having PR statuses notified at all? Since the statuses are not updated accordingly, when corresponding Github Actions finish, it's just unnecessary spam in Slack. And better yet would be if it was possible to turn statuses off completely.

**Comment by vinniefalco** [comment]
Just fix it, don't "look into it." This is interfering with productive technical conversations
![image](https://user-images.githubusercontent.com/1503976/115263055-f1d4a080-a0e9-11eb-85da-21fc20b60250.png)

**Comment by sandersiim** [comment]
☝️ strongly agree.

**Comment by ivokund** [comment]
This has made our github updates channel completely unreadable, really looking forward to a quick fix!

**Comment by vinniefalco** [comment]
Is this integration open source? Is there the possibility of submitting a pull request that adds an option to hide the status messages?

**Comment by ashokirla** [comment]
Apologies for the inconvenience. As we are continuing to work on adding more notification types, the current way of providing strings for each feature type to filter doesn't really scale. We intend to enable more granularity and customizability for the notifications you receive. To achieve that, we need to make this change so that we can standardize on the high level features (issues, prs, commits, releases, actions etc.) and then provide an advanced experience to further customize notifications for each of the features. 

Please bear with us. This is the next priority functionality we will be working on and we sure you would appreciate this feature customizability once it is available. 

Please let me know if you would like to know more about our plan or provide your valuable inputs. I am happy to discuss over a call.

**Comment by vinniefalco** [comment]
It would be nice if those CI status messages were collapsible (like images), and collapsed by default. This wouldn't require any work on the filtering interface (since it wouldn't have any options).

**Comment by Sibi-WC-VW** [comment]
Kindly prioritize it, we would like to filter the PR notifications based on the status

**Comment by ashokirla** [comment]
We have evaluated this feedback. Today, the slack platform doesn't support ability to control when and where to collapse the messages. We have raised a feature request with the slack platform and we will implement this request once it is available.

**Comment by vinniefalco** [comment]
PLEASE help us... we put a man on the moon, can't we somehow inhibit all that output?
![image](https://user-images.githubusercontent.com/1503976/163184399-824dfd44-308f-4668-8084-6f8d52bca48c.png)

**Comment by ashokirla** [comment]
@odinho Yes, as part of Actions notifications, we are ensuring this problem is handled. We will show only the summary in the main card to reduce the clutter and the details of the jobs will be part of the thread. Here is a mock. 
![Thread (1 failed, 1 approval)](https://user-images.githubusercontent.com/25578249/203834196-f0f968df-afe2-4b09-8eca-d8abb11c2675.png)

However, the problem with PR card with checks still needs to be handled. We are waiting for a fix from Slack team as we don't control the UI.

**Comment by odinho** [comment]
> However, the problem with PR card with checks still needs to be handled. We are waiting for a fix from Slack team as we don't control the UI.

Well. For our case, just not having them *at all* on the PR card, or only having a combined one would be very good.

We don't need any extra UI for this. Even what you are doing here showing "extras" in a reply-thread seems to be a usable way to me. So I don't understand why any extra features are needed from Slack.

That just sounds like a recipe for having to wait forever. :)

**Comment by vinniefalco** [comment]
Still a problem unfortunately. 
![image](https://user-images.githubusercontent.com/1503976/204041057-001c669b-8e86-4075-9389-bdbe7194cf17.png)

**Comment by gfpoliva** [comment]
@ashokirla Is there any news? It's been 3 months since dec 6th, which supposed to be the release date.

**Comment by vinniefalco** [comment]
The sad state of it
![image](https://user-images.githubusercontent.com/1503976/223234910-516e8ca9-0b70-4ee2-b2cb-9b044a4ccd56.png)

**Comment by wweaver** [comment]
Still hopeful the workflow status can be hidden in workflows. No way to add configuration to what is shown. I don't mind having failed workflows show, but overall if the choice is to have none or have how it currently works I'd rather not have them.

Skipped is what I really want gone.
<img width="181" alt="image" src="https://github.com/user-attachments/assets/38bbd995-1c61-49b3-b9db-3b9bf9089c8b">

**Comment by vinniefalco** [comment]
Salesforce.
